Tools/markdown-tool/Commands/FormatCommand.swift

```diff
@@ -206,6 +209,10 @@ extension MarkdownCommand {
                 throw ArgumentParser.ValidationError("The value '\(self.emphasisMarker)' is invalid for '--emphasis-marker'")
             }
 
+            guard let strongEmphasisMarker = MarkupFormatter.Options.EmphasisMarker(argument: strongEmphasisMarker ?? self.emphasisMarker) else {
+                throw ArgumentParser.ValidationError("The value '\(self.strongEmphasisMarker)' is invalid for '--strong-emphasis-marker'")
```

Comment:
```
warning: string interpolation produces a debug description for an optional value; did you mean to make this explicit?
```

```suggestion
            if let strongEmphasisMarker {
                guard let strongEmphasisMarker = MarkupFormatter.Options.EmphasisMarker(argument: strongEmphasisMarker) else {
                    throw ArgumentParser.ValidationError("The value '\(self.strongEmphasisMarker)' is invalid for '--strong-emphasis-marker'")
            }
```

We have already provide the fallback logic in `MarkupFormatter.Options`. So I prefer we do not duplicated it again here.

Reply:
I see what you're going for here, but the suggestion actually doesn't build. It's a weird situation because the variable name is being shadowed but needs to be passed into the initializer later in the function

Reply:
Confirm it locally and updated the suggestion.

Reply:
I've pushed a new commit that addresses this. It's not as pretty as I would like, but it works and isn't too ugly.